alacritty_terminal/src/renderer/rects.rs

```diff
@@ -100,10 +126,7 @@ impl RenderLines {
 
             // Check if there's an active line
             if let Some(line) = self.inner.get_mut(flag).and_then(|lines| lines.last_mut()) {
-                if cell.line == line.start.line
-                    && cell.fg == line.color
-                    && cell.column == line.end.col + 1
-                {
+                if cell.fg == line.color && cell.column == line.end.col + 1 {
```

Comment:
Would you mind elaborating on removing `cell.line == line.start.line` check, just for me to faster understand things.

Reply:
I thought that it's not really great that a `RenderLine` which is a `Point -> Point` only supports points on the same line. So the change I've made to the `rects()` function on `RenderLine` is actually responsible itself for converting a `RenderLine` which spans multiple lines into multiple rects.

This nicely extracts the logic of handling multiple lines from this place and the URL handling and places it right at the place closest to conversion.